src/main/resources/drivers/CiscoNext-GenSeries.js

```diff
@@ -0,0 +1,384 @@
+/**
+ * Cisco Catalyst Next-Gen Series (Cisco Business-like CLI) Driver for Netshot
+ * Target: Catalyst 1200 Series (e.g., C1200-16T-2G)
+ *
+ * Author: David Micallef (Pizu)
+ * Version: 1.0
+ */
+
+/* ========================= DRIVER META ========================= */
+
+var Info = {
+  name: "CiscoNextGenSeries",
+  description: "Cisco Catalyst Next-Gen Series",
+  author: "David Micallef (Pizu)",
+  version: "1.0"
+};
+
+/* ========================= CONFIG MAP ========================= */
+
+var Config = {
+  runningConfig: {
+    type: "LongText",
+    title: "Running configuration",
+    comparable: true,
+    searchable: true,
+    checkable: true,
+    dump: {
+      pre: "!! Running configuration (taken on %when%):",
+      post: "!! End of running configuration"
+    }
+  },
+  firmwareVersion: {
+    type: "Text",
+    title: "Firmware version",
+    comparable: true,
+    searchable: true,
+    checkable: true
+  }
+};
+
+// Keep Device empty to avoid redefining built-in attributes (name, family, etc.)
+var Device = {};
+
+/* ========================= CLI LAYER ========================= */
+
+var ANSI = "(?:\\u001b\\[[0-9;]*[A-Za-z])*"; // permissive ANSI eater
+
+var PROMPT_EXEC   = new RegExp("^" + ANSI + ".*>\\s?$", "m");                        // e.g. 'Switch>'
+var PROMPT_ENABLE = new RegExp("^" + ANSI + ".*(?:\\(config[^)]*\\))?#\\s?$", "m");  // e.g. 'Switch#', 'Switch(config)#'
+
+var PAGER_RE = new RegExp(
+  "(?:^|\\r?\\n)" +
+  "(?:" +
+    "\\-\\-More\\-\\-" +              // --More--
+    "|More:\\s*<space>.*" +           // More: <space> to continue, q to quit
+    "|\\[?more\\]?\\s*$" +            // some odd lowercase variants
+  ")",
+  "mi"
+);
+
+var CLI = {
+  ssh: {
+    targets: [
```

Comment:
`targets` is not used in CLI modes. You should rather define Username, Password, etc. as new modes in `CLI` object, and then add them to `options` list of `enable` macro. As in Cisco IOS driver for example.